app/code/Magento/Review/Test/Unit/Model/ReviewSummaryTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Review\Test\Unit\Model;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
Is that suppress warning necessary here? I don't see too many dependencies in this class.

Reply:
You were right. Comment was removed